docs/current_docs/features/debugging.mdx

```diff
@@ -12,6 +12,8 @@ Workflow failures can be both frustrating and lead to wasted resources as the te
 
 Dagger lets users drop in to an interactive shell when a workflow run fails, with all the context at the point of failure. This is similar to a debugger experience, but without needing to set breakpoints explicitly. No changes are required to your code.
 
+## Example broken workflow
```

Comment:
```suggestion
```

There isn't a lot of content until this point so breaking it with a sub-heading doesn't seem necessary.

Reply:
I'd remove this, for the reason mentioned above and also might be good to avoid having the "Example" heading in the same page at two different levels.